miniapps/nurbs/CMakeLists.txt

```diff
@@ -37,6 +37,10 @@ add_mfem_miniapp(nurbs_printfunc
   MAIN nurbs_printfunc.cpp
   LIBRARIES mfem)
 
+add_mfem_miniapp(nurbs_mesh_info
```

Comment:
Also add to `makefile`? It doesn't build with `make`.

Reply:
Done.